src/main/java/io/github/oliviercailloux/y2018/apartments/utils/JsonConvert.java

```diff
@@ -0,0 +1,194 @@
+	 * Read a JSON file which contains an Apartment.
+	 *
+	 * @param jsonPath <i>String</i> the path where the JSON file is located
+	 * @return <i>String</i> containing an Apartment into JSON format
+	 * @throws IOException if the file can't be convert into JSON format.
+	 */
+	public static String readApartmentFromJson(String jsonPath) throws IOException {
+
+		File jsonFile = new File(jsonPath);
+
+		try (BufferedReader jsonFilePath = new BufferedReader(new FileReader(jsonFile))) {
+
+			String jsonLine = jsonFilePath.readLine();
+			String jsonRead = jsonLine;
+
+			while (true) {
+				jsonLine = jsonFilePath.readLine();
+				if (jsonLine == null) {
+					break;
+				}
+				jsonRead = jsonRead + "\n" + jsonLine;
+			}
+
+			jsonFilePath.close();
+			LOGGER.info("Apartment has been read with success");
+
+			return jsonRead;
+		}
+	}
+
+	/**
+	 * Gets the address field from an Address JSON.
+	 *
+	 * @param jsonString <i>String</i> the Address into JSON format
+	 * @return <i>String</i> the address field
+	 */
+	public static String getAddressFromJson(String jsonString) {
+		Jsonb jsonb = JsonbBuilder.create();
+
+		final LinkedHashMap<?, ?> result = jsonb.fromJson(jsonString, LinkedHashMap.class);
+		LOGGER.info("Get address");
+		String address = result.get("address").toString();
+
+		if (address.isEmpty()) {
+			throw new IllegalArgumentException();
+		} else {
+			return result.get("address").toString();
+		}
+	}
+
+	/**
+	 * Converts a JSON expression to an Apartment object.
+	 *
+	 * @param jsonString <i>String</i> the JSON expression to convert into Apartment
+	 *                   object
+	 * @return <i>Apartment</i> the Apartment generated
+	 */
+	public static Apartment jsonToApartment(String jsonString) {
+		Jsonb jsonb = JsonbBuilder.create();
+		LOGGER.info("Create Json builder");
+
+		Apartment.Builder apartBuild = jsonb.fromJson(jsonString, Apartment.Builder.class);
+		LOGGER.info("Apartment created from JSON");
+
+		return apartBuild.build();
+	}
+
+	/**
+	 * Converts a JSON expression to a list of Apartments.
+	 *
+	 * @param jsonString <i>String</i> the JSON expression to convert into a list of
+	 *                   Apartments
+	 * @return <i>ArrayList</i> the list of Apartments created
+	 */
+	public static List<Apartment> jsonToApartments(String jsonString) {
+		List<Apartment.Builder> apartmentsBuild;
+		List<Apartment> apartments = new ArrayList<Apartment>();
+		LOGGER.info("Create ArrayList of Apartment");
+
+		Jsonb jsonb = JsonbBuilder.create();
+		LOGGER.info("Create Json builder");
+
+		apartmentsBuild = jsonb.fromJson(jsonString, new ArrayList<Apartment.Builder>() {
+		}.getClass().getGenericSuperclass());
+
+		for (int i = 0; i < apartmentsBuild.size(); i++) {
+			apartments.add(apartmentsBuild.get(i).build());
+		}
+
+		return apartments;
+	}
+
+	/**
+	 * Converts a list of Apartments to a JSON file with the default path
+	 * APARTMENT_PATH_JSON.
+	 *
+	 * @param a <i>ArrayList</i> object to convert into JSON
```

Comment:
La variable passé est `listApartments`

```suggestion
	 * @param listApartments <code>{@link ArrayList}</code> object to convert into JSON
```